src/utils/local-medical-image-loader.ts

```
@@ -0,0 +1,228 @@
+    };
+  }
+}
+
+export function convertMultiframeImageIds(imageIds: string[]) {
+  const newImageIds: string[] = [];
+  imageIds.forEach((imageId) => {
+    const { imageIdFrameless } = getFrameInformation(imageId);
+    const instanceMetaData = metaData.get("multiframeModule", imageId);
+    if (
+      instanceMetaData &&
+      instanceMetaData.NumberOfFrames &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)